README.md

```diff
@@ -1,10 +1,14 @@
-This workspace contains the crates webpki-roots and webpki-ccadb.
+This workspace contains the crates `webpki-roots`, `webpki-root-certs` and `webpki-ccadb`.
 
-The webpki-roots crate contains Mozilla's root certificates for use with
-the [webpki](https://github.com/rustls/webpki) or
-[rustls](https://github.com/rustls/rustls) crates.
+The `webpki-roots` crate contains Mozilla's trusted root certificates for use with
+the [webpki](https://github.com/rustls/webpki) or [rustls](https://github.com/rustls/rustls) crates.
 
-The webpki-ccadb crate populates the root certificates for the webpki-roots crate
+The `webpki-root-certs` is similar to `webpki-roots`, but for use with other projects
+that require the full self-signed X.509 certificate for each trusted root. This is
+unnecessary overhead for `webpki` and `rustls`, prefer using `webpki-roots` for these
```

Comment:
"this is [..], prefer using .." feels like it lacks an extra verb somewhere.

Reply:
Rewored.